docs/2_1_patching.md

```diff
@@ -13,13 +13,12 @@ The following pages will guide you through using ReVanced Manager to patch apps.
 5. Tap on the **Done** then **Patch** button
    > **Warning**: The patching process may take ~5 minutes. Exiting the app may increase the time it takes to patch.
 6. Tap on the **Install** button
-   > **Note**: If you are rooted, you can mount the patched app on top of the original app.[^4]
-   > Optionally, you may export the patched app to storage in the top right corner.
+   > **Note**: If you are rooted, you can mount the patched app on top of the original app.[^3]
+   > Optionally, you may export the patched app to storage using the options in the top right corner.
 
-[^1]: You may be prompted to select an app from storage in case you have no root access on your device. In this case, select the APK file from your device storage.[^3]
+[^1]: Non-root users may be prompted to select an APK from storage, in which case you have to source the APK file yourself. ReVanced does not provide any APK files.
```

Comment:
The information about having to source APK files yourself should be added to the step where the user needs to select an app. When they already selected one and are already in the installation screen, what use has the footnote about them having to source the APK file.